indico/modules/events/abstracts/controllers/management.py

```diff
@@ -51,6 +51,17 @@ def _process(self):
         else:
             abstracts_count = Abstract.query.with_parent(self.event).count()
             custom_boa = None
+            rules = self.event._get_email_templates_rules()
+            states_string = [str(e.title) for e in AbstractState if e.value not in rules
+                             and e.value in AbstractNotifiedStates]
+            if states_string:
+                if len(states_string) < 1:
+                    states_string = states_string[0]
+                else:
+                    states_string = ', '.join(states_string[:-1]) + ' and ' + states_string[-1]
```

Comment:
This is not i18n friendly.. Maybe you can rephrase it as `Some email notifications are not setup ...`?

Reply:
I would like to show up the notifications are not setup, I believe it is user friendly. Can't we simply translate the and string? the other states are translated properly

Reply:
do we really need this? I think people will make use of this when nothing is set up, but not after partially setting up notifications

Reply:
I would set up this way since people can forget setting up some notifications

Reply:
> Can't we simply translate the and string? the other states are translated properly

It's in general impossible to splice translated sentence fragments and end up with something that makes sense. It might work for languages similar to English but not always. You could perhaps rephrase it as:

```
The following email notifications are not set up: {states}
```
and use babel's `format_list` for the states. But as Adrian says, maybe it's better to only show this if no notifications are set up. I can imagine this warning could be annoying to people who intentionally set up only some notifications.

Reply:
Exactly. If people already started setting up notifications manually they're aware how it works. The main benefit of this whole feature is to quickly get default notifications (which can then be adjusted as needed).

By only doing it when there are no notifications yet, you can probably also simplify the code since you don't need to check which ones exist and which ones don't :)